src/main/java/org/folio/circulation/resources/CheckInProcessAdapter.java

```diff
@@ -102,4 +116,23 @@
 
     return completedFuture(succeeded(item));
   }
+
+  CompletableFuture<HttpResult<CheckInProcessRecords>> sendCheckInPatronNotice(CheckInProcessRecords records) {
```

Comment:
The name of this method refers to a check in notice, yet the invocation within it, is for check out:
`sendCheckOutPatronNoticeWhenPolicyFound(records, policy)`

Which type of notice should this code be sending?

Reply:
The idea of this method is to move processing of found patron notice into separate place.
Probably should consider better naming.

Reply:
@dmytrotur I'm happy with the separation. I just didn't understand the mismatch between check in and check out within it.

Reply:
Oh, sorry, it's just misspelling, I'll fix it.